src/main.cpp

```diff
@@ -937,6 +955,7 @@ QString UsageArguments ( char** argv )
            "  -Q, --qos             set the QoS value. Default is 128. Disable with 0\n"
            "                        (see the Jamulus website to enable QoS on Windows)\n"
            "  -t, --notranslation   disable translation (use English language)\n"
+           "  -6, --enableipv6      enable IPv6 addressing (IPv4 is always enabled)\n"
```

Comment:
I don't think this adds anything at all.  It still requires further explanation as to what "direct connection to client/server" means, so it's useless verbosity.

Reply:
But do we agree that the original proposal needs more clarification?

Reply:
I'm happy with the wording Tony used.  It's consise and says what it does.

Reply:
I still think it's not clear. Probably the question is:
*Does the user need to know that connections to public servers is still IPv4 only?*
If we want to be specific: Yes. If we think it confuses the user: No.
Maybe it's worth mentioning it at least in the release announcement.

Reply:
> Does the user need to know that connections to public servers is still IPv4 only?

Not "public", use the word "registered".  You can register with a directory no one knows about.  That's not particularly public.

To operate the software, no.  They can just select a listed entry from the directory and the software will use the returned IPv4 address.

If you want to use IPv6, you need to:
a) pass "-6" on the command line, thus indicating you understand how to operate with IPv6 addresses, so need no further help
b) enter an IPv6 name or address in the connection dialogue "Server Address", by hand, so you know that you've chosen a specific server rather than a registered one.

As I say, the help text need not say anything else.  This is not the final form for the feature.  It's intended to be somewhat experimental and the documentation should make that clear.  Those who choose to use it should thus be given the credit for understanding what they've chosen to do.